packages/w/wlogout/files/0001-Make-wlogout-stateless.patch

```diff
@@ -0,0 +1,55 @@
+From 6cd5181b92f067e40194ed598bc94d2d0e249b17 Mon Sep 17 00:00:00 2001
+From: Facundo Ciruzzi <ciruzzifacundo@gmail.com>
+Date: Wed, 26 Jun 2024 17:33:21 -0300
+Subject: [PATCH] create_patch
```

Comment:
Please change the subject to match what the file is named. Also, please generate this patch by using `git format-patch --zero-commit --no-signature`